### PR TITLE
Test: Remove production of unneeded blocks

### PR DESCRIPTION
**Pull request by heifner**
Remove extra block production not needed that only slow down test execution.

Partially #264

### CODE REVIEW

File: unittests/eosio_system_tester.hpp [hunk 1]
```diff
@@ -538,7 +538,8 @@ class eosio_system_tester : public T {
             BOOST_REQUIRE_EQUAL( T::success(), regproducer(p) );
          }
       }
-      T::produce_blocks( 250);
+      T::produce_block();
+      T::produce_block(fc::seconds(1000));
```
Comment (greg7mdp) [inline unittests/eosio_system_tester.hpp:542]: Why `1000`? I would have used `125`, but maybe it doesn't matter?

**Review by greg7mdp — APPROVED** [review]

File: unittests/eosio_system_tester.hpp (hunk 1 above)
Comment (heifner) [reply unittests/eosio_system_tester.hpp:542]: 125 would have been fine. Doesn't matter as the performance is the same.

**Review by linh2931 — APPROVED** [review]
Should we keep or make a single test to test producing multiple blocks?

**Comment by heifner** [comment]
> Should we keep or make a single test to test producing multiple blocks?

There are some remaining that produce multiple blocks.

**Comment by ericpassmore** [comment]
Note:start
group: STABILITY
category: TEST
summary: Remove production of unneeded blocks to improve performance of tests. 
Note:end

**Pull request merged**